__test__/setup/setupEnzyme.js

```diff
@@ -0,0 +1,4 @@
+import Enzyme from 'enzyme';
```

Comment:
can we rename the folder `__test__` to `just `test`? or does it need to have the underscores?